.github/workflows/coverage_converter_publish_actions.yml

```
@@ -0,0 +1,148 @@
+name: Coverage Converter Publish Actions
+
+on:
+  pull_request:
+    types: [ opened, edited, synchronize ]
+    paths:
+      - 'metrics/coverage_converter/**'
+  push:
+    branches: master
+    paths:
+      - 'metrics/coverage_converter/**'
+
+jobs:
+  linux-actions:
+    name: Coverage Converter Linux Publish Actions
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          ref: ${{ github.ref }}
+
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: beta
+          version: 2.10.0-110.5.beta
+
+      - name: Get Dependencies
+        run: pub get
+        working-directory: metrics/coverage_converter/
+
+      - name: Run Analyzer
+        working-directory: metrics/coverage_converter/
+        run: dartanalyzer ./
```

We should fail on any warning, so the `dartanalyzer` command should look like this: 
```suggestion
        run: dartanalyzer . --fatal-infos --fatal-warnings
```